python/src/aac/plugins/validators/undefined_fields/_validate_undefined_fields.py

```diff
@@ -35,6 +35,7 @@ def validate_undefined_fields(
     def validate_dict(dict_to_validate: dict) -> None:
         # Due to a bug in how global validations are currently being applied, we need to look up the schema rather
         #   than rely on the schema provided by the validation system.
+        # POPO Update
```

Comment:
Need to put the comment to the same line to remain consistent with the other flagging efforts.

Reply:
I don't like end-of-line comments generally for the reason that they are harder to find without scrolling horizontally (for which I have a severe distaste) but for the sake of expediency I can update my comments.